src/main/java/cst438/hw1/MovieRating.java

```diff
@@ -0,0 +1,85 @@
+package cst438.hw1;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.time.LocalDateTime;
+import java.util.Date;
```

Comment:
Not sure that util.Date is needed for LocalDateTime functionality. Should both give the same information.